app/views/device_stories/_search_form.html.erb

```diff
@@ -1,6 +1,7 @@
 <%= form_tag(device_stories_path, remote: true, :method => "get",
              class: 'navbar-form navbar-left') do %>
   <%= hidden_field_tag :fullscr, params[:fullscr] %>
+  <%= hidden_field_tag :per_page, params[:per_page] %>
```

Comment:
Looks like these don't come through on the dynamic search since we use the input.submit trick to load the new data w/o a page refresh.